schedulelist.h

```diff
@@ -0,0 +1,20 @@
+#ifndef SCHEDULELIST_H
+#define SCHEDULELIST_H
+
+#include <QString>
+
+class ScheduleList
+{
+public:
+    QString date;
+    QString nameOfDay;
+
+    int numOfCouple;
+    QString timeStapOfCouple;
```

Comment:
Typo in timeStapOfCouple - should be timeStampOfCouple. BTW, why couple? Couple, it's like boy and girl or... (you know the next joke) ;)

Reply:
It's the scope of another Pull Request

Reply:
Why you dushnysh?

Reply:
@Gryniam cicho-cicho😂